knowledge-base/grid-inline-edit-form.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Add inline Telerik Form to the the Grid Rows
+description: How to add inline Telerik Form to the Grid Rows.
+type: how-to
+page_title: Add inline Telerik Form to the the Grid Rows
+slug: grid-kb-inline-form
+position: 
+tags: telerik,blazor,grid,inline,form,rows
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Grid for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+I want to add an inline Telerik Form to the Grid rows when I enter edit mode.
```

Comment:
Lately, the KB descriptions have been in the following format

```
Some problem\goal statement.

This KB answers the following questions:
* How to...
* How to...
* How to...
```

This format helps with SEO and also helps users identify that the KB matches their own case. Furthermore, we've adopted this format for our latest KB improvements.

My recommendation is to adopt this format and think how to expand the Description a bit. Here's an example, feel free to add to it as needed.

```
My end users need to be able to submit an inline Form from within a Grid row.

This KB answers the following questions:

* How do I show an inline Form when the user enters edit mode in the Grid?
* How do I embed an editable inline Form inside a Grid row?
* How do I create a cancelable inline Form inside a Grid row?
```

Reply:
I really like the suggestions, I will include them as-is